tests/utils/test_transaction.py

```diff
@@ -18,3 +38,26 @@ def test_transaction_from_function():
     assert x(None) is None
     assert x(42) is None
     assert x(lambda: None).endswith("<lambda>")
+    assert x(my_lambda) == "tests.utils.test_transaction.<lambda>"
+    assert (
+        x(my_partial) == "partial(<function tests.utils.test_transaction.my_partial>)"
+    )
+    assert (
+        x(my_partial_lambda)
+        == "partial(<function tests.utils.test_transaction.<lambda>>)"
+    )
+
+
+@pytest.mark.skipif(sys.version_info < (3, 4))
```

Comment:
You have to give a `reason` here. See for example https://github.com/getsentry/sentry-python/blob/master/tests/integrations/django/test_basic.py#L804

Reply:
Yup, got it when testing locally, didn't push the fix yet